docker-images/artifacts/kafka-thirdparty-libs/3.6.x/pom.xml

```diff
@@ -115,29 +115,40 @@
             <artifactId>opentelemetry-exporter-sender-grpc-managed-channel</artifactId>
             <version>${opentelemetry.version}</version>
             <scope>runtime</scope>
-            <!-- excluded as we end up with conflicting copies in the image detected by find-classpath-collision.sh -->
             <exclusions>
+                <!-- excluded as we end up with conflicting copies in the image detected by find-classpath-collision.sh -->
                 <exclusion>
                     <groupId>com.google.errorprone</groupId>
                     <artifactId>error_prone_annotations</artifactId>
                 </exclusion>
+                <!-- Excluding Guava (which is the android version) because we bring the jre one directly already -->
```

Comment:
Why is this needed? It does not seem to pull Android build of guava.

Reply:
Doesn't pull it in because we have direct guava jre dependency. 
If you run a `mvn dependency:tree -Dverbose` you will get this in a couple of place for otel and grpc:

```shell
(com.google.guava:guava:jar:32.1.3-jre:runtime - version managed from 32.1.3-android; omitted for duplicate)
```

If you by any chance remove the guava jre direct dependency, you pull guava android in.
I think that for clarity it's better having the exclusion so it's clear that for otel and grpc we are relying on the guava jre (which makes more sense compared to the android).

Reply:
Yeah I was trying to make the things clearer, but you raised a good point. I will remove the exclusions, from the corresponding bridge PR as well.